code/_onclick/hud/plane_master.dm

```diff
@@ -27,11 +27,6 @@
 	blend_mode = BLEND_MULTIPLY
 	mouse_opacity = 0
 
-/obj/screen/plane_master/lighting/proc/params2color(params)
-	color = params2list(params)
-/obj/screen/plane_master/lighting/proc/basecolor()
-	color = LIGHTING_BASE_MATRIX
```

Comment:
These procs were removed because they were debug procs i added in another pr and forgot to remove from the committed version.